### ISSUE TITLE
Share box still present during edit of graphical view

### ISSUE THREAD

**Issue by nduvernois** [SCADA-LTS/Scada-LTS#2775]
When a graphical view is edited, a box containing Sharing information shortly appears, and disappears within a second. This is probably a remain of the previous way to determine which user has which rights to see which view. As this is now handled by the user profiles, this "Sharing" box should not appear at all, unless it would allow to select user profiles, which would actually be a good idea.

ScadaLTS 2.6.18
Chrome 118
Windows Server 2016

**Comment by fabiodurao** [comment]
Yes, before the development of the user profile, when creating a new user, there was the option of giving him access to datasources and datapoints, today this is already done in the profile, but this feature is still present when creating a new user.

Same thing with graphical representations as mentioned above, generally it loads and then disappears, for most users it will just be a blink on the screen and those who are less attentive won't notice it, but as I saw mentioned in another issue that it is suffering from slowness in the graphical representation (perhaps it is due to the accumulation of alarms, this has already been resolved in more recent versions) it is now possible to see the share box for a longer time.

If you notice, in the watchlist there is also an icon that appears in the upper right corner, along with the icons for editing the name, duplicating, creating new and deleting watchlist, but again, it only appears and then is "hide" and disappears, too. it is part of the legacy way in which sharing was done with each user. In the past, the biggest problem was that the user who had created the graphical representation or the watchlist was forced to share it with new users, so in large systems with many admin users, this was critical, as they needed to look up which user created the screen to then share with new users, but this no longer happens. Just a little context to understand why the system has these items, but I believe that the redundant and obsolete options will soon be removed.
